packages/components/src/components/form/Select/Select.tsx

```diff
@@ -179,13 +179,24 @@ const Wrapper = styled.div<WrapperProps>`
         display: flex;
         align-items: center;
         justify-content: ${({ $isClean }) => ($isClean ? 'flex-end' : 'flex-start')};
-        width: 100%;
+        width: ${({ $isSearchable }) => !$isSearchable && '100%'};
         max-width: initial;
         color: ${({ $isDisabled, theme }) =>
             $isDisabled ? theme.textDisabled : theme.textDefault};
         border-style: none;
         transform: none;
-        margin-left: 0;
+        margin: 0;
+
+        &:hover {
+            cursor: ${({ $isSearchable }) => $isSearchable && 'text'};
+        }
+    }
+
+    .${reactSelectClassNamePrefix}__single-value + .${reactSelectClassNamePrefix}__input-container {
```

Comment:
@jvaclavik This component is full of the Dark Arts 🔮. WEN floating UI?

Reply:
The sooner, the better. Wen you have free week? 😀

Reply:
Me? You have the biggest team now... :D